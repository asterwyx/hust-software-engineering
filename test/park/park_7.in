preset user AQ
preset gift A robot 5
preset gift A barrier 5
preset userloc A 12 0
preset userloc Q 45 0
preset nextuser Q
step 1 #Q:45-46
n
step 2 #A:12->14 park
step 1 #Q:46->47
n
robot
block 5
block -5
robot
step 1 #A:14->15
n
dump