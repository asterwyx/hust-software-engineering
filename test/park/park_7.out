user AQ
fund A 10000
credit A 0
userloc A 15 0
gift A barrier 3
gift A robot 3
fund Q 10000
credit Q 0
userloc Q 47 0
barrier 9
nextuser Q